apps/backend/src/datasources/FapDataSource.ts

```diff
@@ -76,6 +76,7 @@ export interface FapDataSource {
     callId: number
   ): Promise<BasicUserDetails[]>;
   getFapProposalCount(fapId: number): Promise<number>;
+  getCurrentFapProposalCount(fapId: number): Promise<number>;
```

Comment:
Would it make sense for the new field be renamed to `getFapProposalCountCurrentRound`? That way it'd be a consistent naming scheme with `getFapReviewerProposalCount` and `getFapReviewerProposalCountCurrentRound` below

Reply:
Yeah, we should make them consistent, but I think I am in favour of removing the "round" from  `getFapReviewerProposalCountCurrentRound` as there is no concept of a round in the software only open calls which in theory there could be multiple of going on at once.